package.json

```diff
@@ -49,7 +49,7 @@
     "pull-stream": "^3.7.0",
     "set.prototype.difference": "^1.0.2",
     "ssb-bfe": "^3.7.0",
-    "ssb-box2": "^7.0.0",
+    "ssb-box2": "ssbc/ssb-box2#un-exclude",
```

Comment:
https://github.com/ssbc/ssb-box2/pull/33

Groups weren't showing up in lists because we never un-marked the group as `excluded`. So now we un-mark it when we add a new key to the group.

Reply:
7.1.0